### ISSUE TITLE
Lexica Botania Book Bug

### ISSUE THREAD

**Issue by LinkNightblade** [Vazkii/Botania#2313]
After deleting bookmarks in the Lexica (I had bookmarks for Rune of Fire, Water, Pride, Tektonic Girdle and Ring of Far Reach), clicking the 'Back' arrow at the bottom of the page (or right clicking to return) appeared to close the book and lock me out of doing anything. Pressing Escape put me back into a playable mode however I still couldn't return to the Index/Home page of the Lexica and attempting to do so would continually lock me out.

I was playing Agrarian Skies 2 vr 1.1.14, Botania r1.7-212.

**Comment by LinkNightblade** [comment]
Restarting the client completely fixes this but that is not immediately obvious as reconnecting to the server/game does nothing.

**Comment by Hubry** [comment]
Can you reproduce this in a more recent version? The latest build for 1.7.10 is R1.8-249 (37 builds!), and I'm pretty sure 1.7 is not supported at all - in fact, I don't think 1.10 is supported anymore.

**Comment by williewillus** [comment]
what page were you on when you clicked back?

**Comment by LinkNightblade** [comment]
Hubry, I've tried replicating that in that version but whatever the bug was must not have made it thru or got patched out somehow. I can keep trying though.

Williewillus, I was on one of the recipe pages for a rune in the runic altar section. I believe it was the Fire Rune. Backing out of that page normally was fine, it was after I deleted a bunch of bookmarks, then deleted the one for the fire rune and tried to click back.
